modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/resources/META-INF/resources/deprecated_warning.jspf

```diff
@@ -8,15 +8,10 @@
 <clay:alert
 	displayType="warning"
 >
-	<liferay-learn:message
-		key="dynamic-data-lists-is-deprecated-and-replaced-by-object"
-		resource="dynamic-data-lists-web"
-	/>
-
-	<liferay-ui:message key="ddl-will-be-removed-in-2024" />
+	<liferay-ui:message key="dynamic-data-lists-is-deprecated-and-will-be-removed-in-2024" />
```

Comment:
Since we changed these keys, I believe we also need to change the [description](https://github.com/liferay/liferay-portal/blob/32177bb2ecf85af11f77568de89bc86d7ceb0b0e/modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties#L7297) of the DDL deprecation FF to make it consistent.
I asked in the ticket about this, I think you can wait the answer before send this change.

Reply:
Done! :heavy_check_mark: